### DISCUSSION TITLE
Add group/hardcore group ironman to hiscore lookup

### DISCUSSION THREAD

**Discussion by Solution6996** [runelite/runelite#15044] [Ideas]
Would be nice if is one could look up (HC)GIM in the hiscores while being recognized as ironmen. They're currently only found under 'Normal hiscores' and I personally don't think they should be. 
In terms of implementation I would suggest adding an additional button for GIMs, but you could also implment a rightclick on the iron/hc icons to switch them to GIM lookup to not clutter the UI.

**Comment by kyletherenegade** [comment]
i agree
